components/common/inbox/email-preview.tsx

```
@@ -0,0 +1,111 @@
+                  </Button>
+               )}
+            </div>
+         </div>
+
+         <div className="pt-10 pb-6 px-4 space-y-4 w-full max-w-4xl mx-auto">
+            <div>
+               <h3 className="text-xl font-semibold text-foreground mb-2">{email.title}</h3>
+            </div>
+
+            <div className="prose prose-sm max-w-none">
+               <div dangerouslySetInnerHTML={{ __html: bodyHtml }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)